ibdotxyz/studio
# feat(iron-bank): added iron bank to Zapper

## Description

Adding integration of Iron Bank (ETH, FTM, AVAX)

## Checklist

- [v] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)

## How to test?

1. Checked TVL against Iron Bank's [website](https://app.ib.xyz/markets/Avalanche)
2. Check account balance with a sample from [USDC market on Ethereum](https://etherscan.io/address/0x76Eb2FE28b36B3ee97F3Adae0C69606eeDB2A37c)
3. Check account balance with a sample from [WFTM market on Fantom](https://ftmscan.com/address/0xd528697008ac67a21818751a5e3c58c8dae54696)
4. Check account balance with a sample from [USDT.e market on Avalanche](https://snowtrace.io/address/0xceb1ce674f38398432d20bc8f90345e91ef46fd3)
